README.md

```diff
@@ -10,13 +10,29 @@ Open Source Education Bot built by the Facebook Developer Circles community to h
 
 > **requirement**: python 3.6 or more
 
-Start by cloning the repository locally and enter the project folder into your system.
+Start by  fork and then cloning the repository locally and enter the project folder into your system.
 
 ```bash
-git clone https://github.com/fbdevelopercircles/open-source-edu-bot
+git clone https://github.com/fbdevelopercircles/YOUR_GITHUB_USER_NAME/open-source-edu-bot 
+```
+
+where YOUR_GITHUB_USER_NAME is your GitHub handle.
+
+```
 cd open-source-edu-bot
+git remote add upstream https://github.com/fbdevelopercircles
```

Comment:
```git remote add upstream https://github.com/fbdevelopercircles``` is not working

It should be update as the following:

--
Firstly, you have to use git remote -v for managing set of tracked repositories.You have to add a new remote git remote add upstream https://github.com/fbdevelopercircles/open-source-edu-bot